tests/python-gpu/test_gpu_gdf.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pandas as pd
+import cudf.dataframe as gdf
+from sklearn import datasets
+import sys
+import unittest
+import xgboost as xgb
+
+from regression_test_utilities import run_suite, parameter_combinations, \
+    assert_results_non_increasing, Dataset
+
+
+def get_gdf():
```

Comment:
I am confused by gdf/cudf naming. Are they different things?

Reply:
As I'm sure you're aware, the GDF naming scheme came from our base project from GPU Open Analytics. It stands for GPU DataFrame. cuDF is the name we gave the core library component in RAPIDS.

The initial refactor effort took all of the file names and changed them to cuDF. The data structures, etc., are still named GDF, and this will very likely change to be consistent with the file naming scheme.